tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -35,15 +35,21 @@ def device_name(self):
     def enable_bluetooth(self):
         self.marionette.find_element(*self._bluetooth_label_locator).tap()
```

Comment:
actually checkbox element is outside the menu region, so tapping it won't change the bluetooth status, same for the visible_to_all_checkbox_locator